rascaline/src/calculators/lode/radial_basis.rs

```diff
@@ -66,7 +66,8 @@ impl LodeRadialBasis {
                 let parameters = SplinedRIParameters {
                     max_radial: parameters.max_radial,
                     max_angular: parameters.max_angular,
-                    cutoff: parameters.cutoff,
+                    // The spline cutoff needs to be the cutoff in k-space
+                    cutoff: parameters.get_k_cutoff(),
                 };
```

Comment:
@Luthaf the compiler is blaming me here that `method not found in `GtoParameters` even though we import LodeSphericalExpansionParameters. There is probably something else to import here?

Reply:
Might need to make the function public `pub fn`